### ISSUE TITLE
Can't build project with webpack: Error: Can't resolve './lib-cov/client'

### ISSUE THREAD

**Issue by jsinai** [diversario/node-ssdp#94]
Webpack wants to resolve all dependencies, whether process.env.SSDP_COV is defined or not. In order to get webpack to build my project, I had to do this:

```
cd node_modules/node-ssdp
npm i -D istanbul mocha mocha-istanbul
make coverage
cd -
npx webpack --mode production
```

This is inconvenient, and should not be necessary.

**Comment by diversario** [comment]
@jsinai I've not used webpack but do you think this would make it deal with it better:

```
module.exports = {
  Server: require(process.env.SSDP_COV ? "./lib/server" : "./lib-cov/server"),
  Client: require(process.env.SSDP_COV ? "./lib/client" : "./lib-cov/client"),
  Base: require(process.env.SSDP_COV ? "./lib/index" : "./lib-cov/index")
}
```
since it sounds like the issue is with using `require`?

**Comment by diversario** [comment]
Published v4.0.0. Give it a try!

**Comment by jsinai** [comment]
Sorry for the very late response, but your fix worked.
